pkg/feature/features.go

```diff
@@ -40,6 +40,11 @@ const (
 	// ExperimentalGatewayAPISupport enables the gateway-shim controller and adds support for
 	// the Gateway API to the HTTP-01 challenge solver.
 	ExperimentalGatewayAPISupport featuregate.Feature = "ExperimentalGatewayAPISupport"
+
+	// alpha: v1.7.0
+	//
+	// ExperimentalCertificatePrivateKeyAdditionalFormats enable private key additional format
+	ExperimentalCertificatePrivateKeyAdditionalFormats featuregate.Feature = "ExperimentalCertificatePrivateKeyAdditionalFormats"
```

Comment:
what dou you mean by graduation path ?

Reply:
Despite the others _not_ being named accordingly, I think we're better to not have `Experimental` in the name here. Maybe just `AdditionalCertificateOutputFormats`? (given that we're expecting this feature to also cover more than just private keys too).

> what dou you mean by graduation path ?

I think Josh refers to how we make this feature go from alpha->beta->GA (and as part of that, enable the feature by default). I think this is a good one to discuss at the biweekly community meeting on Wednesday's. It may just be that we note someone here as an 'owner' of the feature, and that person is responsible for 'pushing the feature forward' once this PR has merged and the feature has been rolled out to users :)

Reply:
OK, So I've renamed the feature to `AdditionalCertificateOutputFormats`. I'll try to connect at the next community meeting to discuss about the graduation path.